### DISCUSSION TITLE
Adding a Banner above the whole NavMenu

### DISCUSSION THREAD

**Discussion by fdevans** [vuepress-theme-hope/vuepress-theme-hope#4225] [Q&A]
Hi Hope Theme Friends,
  I have been trying to implement a banner across the full top of our page to be used for annoucements.  I have done some work with the Slots and understand how to get content into those from the documentation.  My question is if there is a slot, or other approach I could use to create a banner similar to what is shown below in the green box.

  
<img width="1170" alt="Screenshot 2024-06-13 at 11 58 46 AM" src="https://github.com/vuepress-theme-hope/vuepress-theme-hope/assets/9272445/19cb8dc4-ce7b-4f70-9e77-2e2635087a9e">

**Comment by Mister-Hope** [comment]
May be over riding the navbar component

**Comment by Mister-Hope** [comment]
I think the easier approach might be transforming div#app down a little.bit

**Comment by fdevans** [comment]
Thanks for the suggestions.  We ended up placing the annoucements in other areas using the "Slots" placement here:

https://theme-hope.vuejs.press/guide/advanced/replace.html#using-slots

This worked great for our needs.
